### ISSUE TITLE
[bug] Approval request can be delivered not instanly for locked Misskey (or it fork) account

### ISSUE THREAD

**Issue by SadmL** [superseriousbusiness/gotosocial#3523]
### Describe the bug with a clear and concise description of what the bug is.

Just find out that approval request for locked Misskey or Sharkey (tested on them) account can be delivered after some time or only after several attempts, or not delivered at all, or mix of all 3 of this. Waiting time is up to 2 mins.
I don't think that happens due lack of stable connection, at least on my side, I have 300MiB bandwidth. Also it was tested on 2 separate servers that geographically in different parts of country (may be I'm very lucky 🙃).

I've tried to expire keys via admin menu, but no effect.

But it's almost instant if this Misskey (or it fork) account is following you first.

I've tried Moshidon, Mastodon official app, Phanpy, Elk - result near same.

### What's your GoToSocial Version?

v0.17.3

### GoToSocial Arch

amd64 Binary

### What happened?

<details><summary>Logs</summary>
<p>

```
ноя 06 21:01:09 server gotosocial[129664]: timestamp="06/11/2024 21:01:09.643" func=server.init.func1.Logger.13.1 level=INFO latency="50.492645ms" userAgent=MoshidonAndroid/2.3.0+fork.107.moshinda-nightly+@2024-09-09 method=GET statusCode=200 path=/api/v1/accounts/01J6HXVT8RRK24YBSRZCT8B3E4 clientIP=192.168.50.1 requestID=v0ra60mk04000w1q94rg msg="OK: wrote 1.26kiB"
ноя 06 21:01:09 server gotosocial[129664]: timestamp="06/11/2024 21:01:09.706" func=server.init.func1.Logger.13.1 level=INFO latency="21.011265ms" userAgent=MoshidonAndroid/2.3.0+fork.107.moshinda-nightly+@2024-09-09 method=GET statusCode=200 path=/api/v1/accounts/01J6HXVT8RRK24YBSRZCT8B3E4/statuses clientIP=192.168.50.1 requestID=6mrt60mk04001e3g792g msg="OK: wrote 2.63kiB"
ноя 06 21:01:09 server gotosocial[129664]: timestamp="06/11/2024 21:01:09.716" func=server.init.func1.Logger.13.1 level=INFO latency="30.669258ms" userAgent=MoshidonAndroid/2.3.0+fork.107.moshinda-nightly+@2024-09-09 method=GET statusCode=200 path=/api/v1/accounts/relationships clientIP=192.168.50.1 requestID=6rrt60mk04000chs6ar0 msg="OK: wrote 176B"
ноя 06 21:01:09 server gotosocial[129664]: timestamp="06/11/2024 21:01:09.728" func=server.init.func1.Logger.13.1 level=INFO latency="8.501044ms" userAgent=MoshidonAndroid/2.3.0+fork.107.moshinda-nightly+@2024-09-09 method=GET statusCode=200 path=/api/v1/accounts/01J6HXVT8RRK24YBSRZCT8B3E4/statuses clientIP=192.168.50.1 requestID=awrt60mk04001f36pnbg msg="OK: wrote 26B"
ноя 06 21:01:09 server gotosocial[129664]: timestamp="06/11/2024 21:01:09.737" func=server.init.func1.Logger.13.1 level=INFO latency="51.710481ms" userAgent=MoshidonAndroid/2.3.0+fork.107.moshinda-nightly+@2024-09-09 method=GET statusCode=200 path=/api/v1/accounts/01J6HXVT8RRK24YBSRZCT8B3E4/statuses clientIP=192.168.50.1 requestID=6mrt60mk04000y4x1h1g msg="OK: wrote 1.97kiB"
ноя 06 21:01:13 server gotosocial[129664]: timestamp="06/11/2024 21:01:13.131" func=server.init.func1.Logger.13.1 level=INFO latency="4.175754ms" userAgent=MoshidonAndroid/2.3.0+fork.107.moshinda-nightly+@2024-09-09 method=POST statusCode=200 path=/api/v1/accounts/01J6HXVT8RRK24YBSRZCT8B3E4/follow clientIP=192.168.50.1 requestID=mwza60mk04000kw7qcv0 msg="OK: wrote 179B"
ноя 06 21:01:13 server gotosocial[129664]: timestamp="06/11/2024 21:01:13.131" func=workers.(*Processor).ProcessFromClientAPI level=INFO activityType=Create objectType=Follow fromAccount=sl msg="processing from client API"
ноя 06 21:01:13 server gotosocial[129664]: timestamp="06/11/2024 21:01:13.134" func=federation.(*federatingActor).Send level=INFO msg="send activity Follow via outbox https://gts.sadium.cyou/users/sl/outbox"
ноя 06 21:01:13 server gotosocial[129664]: timestamp="06/11/2024 21:01:13.370" func=httpclient.(*Client).DoOnce level=INFO method=POST url=https://shitpost.poridge.club/inbox contentType="application/ld+json; profile=\"https://www.w3.org/ns/activitystreams\"" attempt="1" pubKeyID=https://gts.sadium.cyou/users/sl/main-key msg="202 Accepted"
ноя 06 21:01:13 server gotosocial[129664]: timestamp="06/11/2024 21:01:13.391" func=server.init.func1.Logger.13.1 level=INFO latency="19.210209ms" userAgent="Misskey/2024.10.1 (https://shitpost.poridge.club/)" method=GET statusCode=200 path=/users/sl clientIP=178.208.94.78 pubKeyID=https://shitpost.poridge.club/users/8qtra2s8ub#main-key requestID=kgzt60mk04000a1byrj0 msg="OK: wrote 2.13kiB"
ноя 06 21:01:13 server gotosocial[129664]: timestamp="06/11/2024 21:01:13.486" func=server.init.func1.Logger.13.1 level=INFO latency="978.13µs" userAgent="Misskey/2024.10.1 (https://shitpost.poridge.club/)" method=GET statusCode=200 path=/users/sl/following clientIP=178.208.94.78 pubKeyID=https://shitpost.poridge.club/users/8qtra2s8ub#main-key requestID=1n0a60mk0400019d6pcg msg="OK: wrote 156B"
ноя 06 21:01:13 server gotosocial[129664]: timestamp="06/11/2024 21:01:13.497" func=server.init.func1.Logger.13.1 level=INFO latency="12.076006ms" userAgent="Misskey/2024.10.1 (https://shitpost.poridge.club/)" method=GET statusCode=200 path=/users/sl/followers clientIP=178.208.94.78 pubKeyID=https://shitpost.poridge.club/users/8qtra2s8ub#main-key requestID=1n0a60mk040002hyt8z0 msg="OK: wrote 157B"
ноя 06 21:01:14 server gotosocial[129664]: timestamp="06/11/2024 21:01:14.337" func=server.init.func1.Logger.13.1 level=INFO latency="770.368656ms" userAgent="Misskey/2024.10.1 (https://shitpost.poridge.club/)" method=GET statusCode=200 path=/fileserver/01QDFX9DAY02T3A3CJW67MB94V/attachment/original/01J5BW8XZWRZMDB36PX8N4X3QX.jpg clientIP=178.208.94.78 requestID=bs0a60mk04000keyrr80 msg="OK: wrote 696kiB"
ноя 06 21:01:14 server gotosocial[129664]: timestamp="06/11/2024 21:01:14.341" func=server.init.func1.Logger.13.1 level=INFO latency="774.822052ms" userAgent="Misskey/2024.10.1 (https://shitpost.poridge.club/)" method=GET statusCode=200 path=/fileserver/01QDFX9DAY02T3A3CJW67MB94V/attachment/original/01JBYDNDZV8XKR5JVCNNH8J1MX.jpeg clientIP=178.208.94.78 requestID=bs0a60mk04000namxw60 msg="OK: wrote 266kiB"
ноя 06 21:01:20 server gotosocial[129664]: timestamp="06/11/2024 21:01:20.479" func=streaming.(*Module).handleWSConn level=INFO streamID=01JC1A6PTY6DRDC0ESM58JA3X1 username=sl requestID=bddt60mk04000497fqw0 msg="opened websocket connection"
ноя 06 21:01:20 server gotosocial[129664]: 2024/11/06 21:01:20 http: response.Write on hijacked connection from github.com/gin-gonic/gin.(*responseWriter).Write (response_writer.go:83)
ноя 06 21:01:20 server gotosocial[129664]: timestamp="06/11/2024 21:01:20.481" func=server.init.func1.Logger.13.1 level=INFO latency="6.145083ms" userAgent="Mozilla/5.0 (X11; Linux x86_64; rv:132.0) Gecko/20100101 Firefox/132.0" method=GET statusCode=200 path=/api/v1/streaming clientIP=192.168.50.1 requestID=bddt60mk04000497fqw0 msg="OK: wrote 0B"
ноя 06 21:01:20 server gotosocial[129664]: timestamp="06/11/2024 21:01:20.482" func=streaming.(*Module).readFromWSConn level=INFO streamID=01JC1A6PTY6DRDC0ESM58JA3X1 username=sl requestID=bddt60mk04000497fqw0 msg="received websocket message: {Type:subscribe Stream:user:notification List:}"
ноя 06 21:01:20 server gotosocial[129664]: timestamp="06/11/2024 21:01:20.482" func=streaming.(*Module).readFromWSConn level=WARN streamID=01JC1A6PTY6DRDC0ESM58JA3X1 username=sl requestID=bddt60mk04000497fqw0 msg="unknown 'stream' field: {subscribe user:notification }"
ноя 06 21:01:20 server gotosocial[129664]: timestamp="06/11/2024 21:01:20.482" func=streaming.(*Module).readFromWSConn level=INFO streamID=01JC1A6PTY6DRDC0ESM58JA3X1 username=sl requestID=bddt60mk04000497fqw0 msg="received websocket message: {Type:subscribe Stream:user:notification List:}"
ноя 06 21:01:20 server gotosocial[129664]: timestamp="06/11/2024 21:01:20.482" func=streaming.(*Module).readFromWSConn level=WARN streamID=01JC1A6PTY6DRDC0ESM58JA3X1 username=sl requestID=bddt60mk04000497fqw0 msg="unknown 'stream' field: {subscribe user:notification }"
ноя 06 21:01:20 server gotosocial[129664]: timestamp="06/11/2024 21:01:20.482" func=streaming.(*Module).readFromWSConn level=INFO streamID=01JC1A6PTY6DRDC0ESM58JA3X1 username=sl requestID=bddt60mk04000497fqw0 msg="received websocket message: {Type:subscribe Stream:user:notification List:}"
ноя 06 21:01:20 server gotosocial[129664]: timestamp="06/11/2024 21:01:20.482" func=streaming.(*Module).readFromWSConn level=WARN streamID=01JC1A6PTY6DRDC0ESM58JA3X1 username=sl requestID=bddt60mk04000497fqw0 msg="unknown 'stream' field: {subscribe user:notification }"
ноя 06 21:01:21 server gotosocial[129664]: 2024/11/06 21:01:21 http: response.Write on hijacked connection from github.com/gin-gonic/gin.(*responseWriter).Write (response_writer.go:83)
ноя 06 21:01:21 server gotosocial[129664]: timestamp="06/11/2024 21:01:21.035" func=streaming.(*Module).handleWSConn level=INFO streamID=01JC1A6QCBKCGDAQE7EP627VD5 username=sl requestID=h9et60mk04000xzrn4t0 msg="opened websocket connection"
ноя 06 21:01:21 server gotosocial[129664]: timestamp="06/11/2024 21:01:21.035" func=server.init.func1.Logger.13.1 level=INFO latency="1.159136ms" userAgent="Mozilla/5.0 (X11; Linux x86_64; rv:132.0) Gecko/20100101 Firefox/132.0" method=GET statusCode=200 path=/api/v1/streaming clientIP=192.168.50.1 requestID=h9et60mk04000xzrn4t0 msg="OK: wrote 0B"
ноя 06 21:01:35 server gotosocial[129664]: timestamp="06/11/2024 21:01:35.176" func=server.init.func1.Logger.13.1 level=INFO latency="3.362552ms" userAgent="Misskey/2024.10.1 (https://shitpost.poridge.club/)" method=GET statusCode=200 path=/users/sl clientIP=178.208.94.78 pubKeyID=https://shitpost.poridge.club/users/8qtra2s8ub#main-key requestID=rjaa60mk040006vkbjsg msg="OK: wrote 2.13kiB"
ноя 06 21:01:35 server gotosocial[129664]: timestamp="06/11/2024 21:01:35.275" func=server.init.func1.Logger.13.1 level=INFO latency="633.733µs" userAgent="Misskey/2024.10.1 (https://shitpost.poridge.club/)" method=GET statusCode=200 path=/users/sl/following clientIP=178.208.94.78 pubKeyID=https://shitpost.poridge.club/users/8qtra2s8ub#main-key requestID=5aat60mk04001k0x50jg msg="OK: wrote 156B"
ноя 06 21:01:35 server gotosocial[129664]: timestamp="06/11/2024 21:01:35.275" func=server.init.func1.Logger.13.1 level=INFO latency="581.272µs" userAgent="Misskey/2024.10.1 (https://shitpost.poridge.club/)" method=GET statusCode=200 path=/users/sl/followers clientIP=178.208.94.78 pubKeyID=https://shitpost.poridge.club/users/8qtra2s8ub#main-key requestID=5eat60mk04000nk8eafg msg="OK: wrote 157B"
ноя 06 21:01:35 server gotosocial[129664]: timestamp="06/11/2024 21:01:35.346" func=server.init.func1.Logger.13.1 level=INFO latency="358.272µs" userAgent="Misskey/2024.10.1 (https://shitpost.poridge.club/)" method=GET statusCode=200 path=/fileserver/01QDFX9DAY02T3A3CJW67MB94V/attachment/original/01JBYDNDZV8XKR5JVCNNH8J1MX.jpeg clientIP=178.208.94.78 requestID=eaat60mk040009pfrmw0 msg="OK: wrote 266kiB"
ноя 06 21:01:36 server gotosocial[129664]: timestamp="06/11/2024 21:01:36.779" func=server.init.func1.Logger.13.1 level=INFO latency="1.433016417s" userAgent="Misskey/2024.10.1 (https://shitpost.poridge.club/)" method=GET statusCode=200 path=/fileserver/01QDFX9DAY02T3A3CJW67MB94V/attachment/original/01J5BW8XZWRZMDB36PX8N4X3QX.jpg clientIP=178.208.94.78 requestID=eaat60mk04001avdgdg0 msg="OK: wrote 696kiB"
ноя 06 21:02:05 server gotosocial[129664]: timestamp="06/11/2024 21:02:05.566" func=workers.(*Processor).ProcessFromClientAPI level=INFO activityType=Undo objectType=Follow fromAccount=sl msg="processing from client API"
ноя 06 21:02:05 server gotosocial[129664]: timestamp="06/11/2024 21:02:05.569" func=federation.(*federatingActor).Send level=INFO msg="send activity Undo via outbox https://gts.sadium.cyou/users/sl/outbox"
ноя 06 21:02:05 server gotosocial[129664]: timestamp="06/11/2024 21:02:05.572" func=server.init.func1.Logger.13.1 level=INFO latency="15.07272ms" userAgent=MoshidonAndroid/2.3.0+fork.107.moshinda-nightly+@2024-09-09 method=POST statusCode=200 path=/api/v1/accounts/01J6HXVT8RRK24YBSRZCT8B3E4/unfollow clientIP=192.168.50.1 requestID=em5t80mk04001yrg686g msg="OK: wrote 173B"
ноя 06 21:02:05 server gotosocial[129664]: timestamp="06/11/2024 21:02:05.622" func=httpclient.(*Client).DoOnce level=INFO method=POST url=https://shitpost.poridge.club/inbox contentType="application/ld+json; profile=\"https://www.w3.org/ns/activitystreams\"" attempt="1" pubKeyID=https://gts.sadium.cyou/users/sl/main-key msg="202 Accepted"
ноя 06 21:02:08 server gotosocial[129664]: timestamp="06/11/2024 21:02:08.163" func=workers.(*Processor).ProcessFromClientAPI level=INFO activityType=Create objectType=Follow fromAccount=sl msg="processing from client API"
ноя 06 21:02:08 server gotosocial[129664]: timestamp="06/11/2024 21:02:08.163" func=server.init.func1.Logger.13.1 level=INFO latency="13.323766ms" userAgent=MoshidonAndroid/2.3.0+fork.107.moshinda-nightly+@2024-09-09 method=POST statusCode=200 path=/api/v1/accounts/01J6HXVT8RRK24YBSRZCT8B3E4/follow clientIP=192.168.50.1 requestID=jrat80mk04001wczex70 msg="OK: wrote 179B"
ноя 06 21:02:08 server gotosocial[129664]: timestamp="06/11/2024 21:02:08.165" func=federation.(*federatingActor).Send level=INFO msg="send activity Follow via outbox https://gts.sadium.cyou/users/sl/outbox"
ноя 06 21:02:08 server gotosocial[129664]: timestamp="06/11/2024 21:02:08.209" func=httpclient.(*Client).DoOnce level=INFO method=POST url=https://shitpost.poridge.club/inbox contentType="application/ld+json; profile=\"https://www.w3.org/ns/activitystreams\"" attempt="1" pubKeyID=https://gts.sadium.cyou/users/sl/main-key msg="202 Accepted"
ноя 06 21:02:08 server gotosocial[129664]: timestamp="06/11/2024 21:02:08.237" func=server.init.func1.Logger.13.1 level=INFO latency="8.040585ms" userAgent="Misskey/2024.10.1 (https://shitpost.poridge.club/)" method=GET statusCode=200 path=/users/sl clientIP=178.208.94.78 pubKeyID=https://shitpost.poridge.club/users/8qtra2s8ub#main-key requestID=wmat80mk04001bm3z0jg msg="OK: wrote 2.13kiB"
ноя 06 21:02:08 server gotosocial[129664]: timestamp="06/11/2024 21:02:08.330" func=server.init.func1.Logger.13.1 level=INFO latency="2.484527ms" userAgent="Misskey/2024.10.1 (https://shitpost.poridge.club/)" method=GET statusCode=200 path=/users/sl/followers clientIP=178.208.94.78 pubKeyID=https://shitpost.poridge.club/users/8qtra2s8ub#main-key requestID=90ba80mk04000p7jth00 msg="OK: wrote 157B"
ноя 06 21:02:08 server gotosocial[129664]: timestamp="06/11/2024 21:02:08.330" func=server.init.func1.Logger.13.1 level=INFO latency="2.588629ms" userAgent="Misskey/2024.10.1 (https://shitpost.poridge.club/)" method=GET statusCode=200 path=/users/sl/following clientIP=178.208.94.78 pubKeyID=https://shitpost.poridge.club/users/8qtra2s8ub#main-key requestID=90ba80mk04001zzeyd10 msg="OK: wrote 156B"
ноя 06 21:02:08 server gotosocial[129664]: timestamp="06/11/2024 21:02:08.376" func=server.init.func1.Logger.13.1 level=INFO latency="862.518µs" userAgent="Misskey/2024.10.1 (https://shitpost.poridge.club/)" method=GET statusCode=200 path=/fileserver/01QDFX9DAY02T3A3CJW67MB94V/attachment/original/01JBYDNDZV8XKR5JVCNNH8J1MX.jpeg clientIP=178.208.94.78 requestID=ewba80mk040004yyhzxg msg="OK: wrote 266kiB"
ноя 06 21:02:08 server gotosocial[129664]: timestamp="06/11/2024 21:02:08.377" func=server.init.func1.Logger.13.1 level=INFO latency="1.056308ms" userAgent="Misskey/2024.10.1 (https://shitpost.poridge.club/)" method=GET statusCode=200 path=/fileserver/01QDFX9DAY02T3A3CJW67MB94V/attachment/original/01J5BW8XZWRZMDB36PX8N4X3QX.jpg clientIP=178.208.94.78 requestID=f0ba80mk04001zew4510 msg="OK: wrote 696kiB"
ноя 06 21:02:18 server gotosocial[129664]: timestamp="06/11/2024 21:02:18.004" func=server.init.func1.Logger.13.1 level=INFO latency="1.130391ms" userAgent=MoshidonAndroid/2.3.0+fork.107.moshinda-nightly+@2024-09-09 method=GET statusCode=200 path=/api/v1/accounts/01J6HXVT8RRK24YBSRZCT8B3E4 clientIP=192.168.50.1 requestID=2cya80mk04001ke3gjf0 msg="OK: wrote 1.26kiB"
ноя 06 21:02:18 server gotosocial[129664]: timestamp="06/11/2024 21:02:18.027" func=server.init.func1.Logger.13.1 level=INFO latency="175.13µs" userAgent=MoshidonAndroid/2.3.0+fork.107.moshinda-nightly+@2024-09-09 method=GET statusCode=200 path=/api/v1/accounts/relationships clientIP=192.168.50.1 requestID=58ya80mk0400122jbyq0 msg="OK: wrote 182B"
ноя 06 21:02:18 server gotosocial[129664]: timestamp="06/11/2024 21:02:18.037" func=server.init.func1.Logger.13.1 level=INFO latency="6.849637ms" userAgent=MoshidonAndroid/2.3.0+fork.107.moshinda-nightly+@2024-09-09 method=GET statusCode=200 path=/api/v1/accounts/01J6HXVT8RRK24YBSRZCT8B3E4/statuses clientIP=192.168.50.1 requestID=5rya80mk04001hdqsc0g msg="OK: wrote 2.63kiB"
ноя 06 21:02:18 server gotosocial[129664]: timestamp="06/11/2024 21:02:18.050" func=server.init.func1.Logger.13.1 level=INFO latency="2.303296ms" userAgent=MoshidonAndroid/2.3.0+fork.107.moshinda-nightly+@2024-09-09 method=GET statusCode=200 path=/api/v1/accounts/01J6HXVT8RRK24YBSRZCT8B3E4/statuses clientIP=192.168.50.1 requestID=80ya80mk04000w6rxka0 msg="OK: wrote 1.97kiB"
ноя 06 21:02:18 server gotosocial[129664]: timestamp="06/11/2024 21:02:18.082" func=server.init.func1.Logger.13.1 level=INFO latency="595.66µs" userAgent=MoshidonAndroid/2.3.0+fork.107.moshinda-nightly+@2024-09-09 method=GET statusCode=200 path=/api/v1/accounts/01J6HXVT8RRK24YBSRZCT8B3E4/statuses clientIP=192.168.50.1 requestID=c4ya80mk04000y407ktg msg="OK: wrote 26B"
ноя 06 21:02:22 server gotosocial[129664]: timestamp="06/11/2024 21:02:22.675" func=workers.(*Processor).ProcessFromClientAPI level=INFO activityType=Undo objectType=Follow fromAccount=sl msg="processing from client API"
ноя 06 21:02:22 server gotosocial[129664]: timestamp="06/11/2024 21:02:22.676" func=server.init.func1.Logger.13.1 level=INFO latency="1.07497ms" userAgent=MoshidonAndroid/2.3.0+fork.107.moshinda-nightly+@2024-09-09 method=POST statusCode=200 path=/api/v1/accounts/01J6HXVT8RRK24YBSRZCT8B3E4/unfollow clientIP=192.168.50.1 requestID=ad7a80mk0400178f0ks0 msg="OK: wrote 173B"
ноя 06 21:02:22 server gotosocial[129664]: timestamp="06/11/2024 21:02:22.700" func=federation.(*federatingActor).Send level=INFO msg="send activity Undo via outbox https://gts.sadium.cyou/users/sl/outbox"
ноя 06 21:02:22 server gotosocial[129664]: timestamp="06/11/2024 21:02:22.747" func=httpclient.(*Client).DoOnce level=INFO method=POST url=https://shitpost.poridge.club/inbox contentType="application/ld+json; profile=\"https://www.w3.org/ns/activitystreams\"" attempt="1" pubKeyID=https://gts.sadium.cyou/users/sl/main-key msg="202 Accepted"
ноя 06 21:02:23 server gotosocial[129664]: timestamp="06/11/2024 21:02:23.906" func=server.init.func1.Logger.13.1 level=INFO latency="25.08901ms" userAgent="python-httpx/0.24.1 (Takahe/0.9.0; +https://social.codespawn.space/)" method=GET statusCode=200 path=/users/sl/statuses/01JC06X4332WQTE57AJKZ4ZPZ7 clientIP=217.174.8.232 pubKeyID=https://social.codespawn.space/actor/#main-key requestID=159t80mk04000yphtzbg msg="OK: wrote 860B"
ноя 06 21:02:24 server gotosocial[129664]: timestamp="06/11/2024 21:02:24.557" func=server.init.func1.Logger.13.1 level=INFO latency="4.344275ms" userAgent="Misskey/2024.10.1 (https://shitpost.poridge.club/)" method=GET statusCode=200 path=/users/sl clientIP=178.208.94.78 pubKeyID=https://shitpost.poridge.club/users/8qtra2s8ub#main-key requestID=n5at80mk04000jbhvdt0 msg="OK: wrote 2.13kiB"
ноя 06 21:02:24 server gotosocial[129664]: timestamp="06/11/2024 21:02:24.643" func=server.init.func1.Logger.13.1 level=INFO latency="661.795µs" userAgent="Misskey/2024.10.1 (https://shitpost.poridge.club/)" method=GET statusCode=200 path=/users/sl/followers clientIP=178.208.94.78 pubKeyID=https://shitpost.poridge.club/users/8qtra2s8ub#main-key requestID=09ba80mk04001s48g62g msg="OK: wrote 157B"
ноя 06 21:02:24 server gotosocial[129664]: timestamp="06/11/2024 21:02:24.643" func=server.init.func1.Logger.13.1 level=INFO latency="593.052µs" userAgent="Misskey/2024.10.1 (https://shitpost.poridge.club/)" method=GET statusCode=200 path=/users/sl/following clientIP=178.208.94.78 pubKeyID=https://shitpost.poridge.club/users/8qtra2s8ub#main-key requestID=09ba80mk0400155s8yv0 msg="OK: wrote 156B"
ноя 06 21:02:24 server gotosocial[129664]: timestamp="06/11/2024 21:02:24.687" func=server.init.func1.Logger.13.1 level=INFO latency="365.698µs" userAgent="Misskey/2024.10.1 (https://shitpost.poridge.club/)" method=GET statusCode=200 path=/fileserver/01QDFX9DAY02T3A3CJW67MB94V/attachment/original/01JBYDNDZV8XKR5JVCNNH8J1MX.jpeg clientIP=178.208.94.78 requestID=5xba80mk040006gvz7kg msg="OK: wrote 266kiB"
ноя 06 21:02:24 server gotosocial[129664]: timestamp="06/11/2024 21:02:24.689" func=server.init.func1.Logger.13.1 level=INFO latency="412.265µs" userAgent="Misskey/2024.10.1 (https://shitpost.poridge.club/)" method=GET statusCode=200 path=/fileserver/01QDFX9DAY02T3A3CJW67MB94V/attachment/original/01J5BW8XZWRZMDB36PX8N4X3QX.jpg clientIP=178.208.94.78 requestID=61ba80mk04000q04zddg msg="OK: wrote 696kiB"
ноя 06 21:02:24 server gotosocial[129664]: timestamp="06/11/2024 21:02:24.956" func=workers.(*Processor).ProcessFromClientAPI level=INFO activityType=Create objectType=Follow fromAccount=sl msg="processing from client API"
ноя 06 21:02:24 server gotosocial[129664]: timestamp="06/11/2024 21:02:24.956" func=server.init.func1.Logger.13.1 level=INFO latency="719.46µs" userAgent=MoshidonAndroid/2.3.0+fork.107.moshinda-nightly+@2024-09-09 method=POST statusCode=200 path=/api/v1/accounts/01J6HXVT8RRK24YBSRZCT8B3E4/follow clientIP=192.168.50.1 requestID=7dbt80mk04000gjze0s0 msg="OK: wrote 179B"
ноя 06 21:02:24 server gotosocial[129664]: timestamp="06/11/2024 21:02:24.956" func=federation.(*federatingActor).Send level=INFO msg="send activity Follow via outbox https://gts.sadium.cyou/users/sl/outbox"
ноя 06 21:02:25 server gotosocial[129664]: timestamp="06/11/2024 21:02:25.001" func=httpclient.(*Client).DoOnce level=INFO method=POST url=https://shitpost.poridge.club/inbox contentType="application/ld+json; profile=\"https://www.w3.org/ns/activitystreams\"" attempt="1" pubKeyID=https://gts.sadium.cyou/users/sl/main-key msg="202 Accepted"
ноя 06 21:02:25 server gotosocial[129664]: timestamp="06/11/2024 21:02:25.044" func=server.init.func1.Logger.13.1 level=INFO latency="1.35667ms" userAgent="Misskey/2024.10.1 (https://shitpost.poridge.club/)" method=GET statusCode=200 path=/users/sl clientIP=178.208.94.78 pubKeyID=https://shitpost.poridge.club/users/8qtra2s8ub#main-key requestID=jdbt80mk04001tx4a3cg msg="OK: wrote 2.13kiB"
ноя 06 21:02:25 server gotosocial[129664]: timestamp="06/11/2024 21:02:25.132" func=server.init.func1.Logger.13.1 level=INFO latency="748.382µs" userAgent="Misskey/2024.10.1 (https://shitpost.poridge.club/)" method=GET statusCode=200 path=/users/sl/following clientIP=178.208.94.78 pubKeyID=https://shitpost.poridge.club/users/8qtra2s8ub#main-key requestID=xdbt80mk04001kzxb5vg msg="OK: wrote 156B"
ноя 06 21:02:25 server gotosocial[129664]: timestamp="06/11/2024 21:02:25.217" func=server.init.func1.Logger.13.1 level=INFO latency="674.892µs" userAgent="Misskey/2024.10.1 (https://shitpost.poridge.club/)" method=GET statusCode=200 path=/users/sl/followers clientIP=178.208.94.78 pubKeyID=https://shitpost.poridge.club/users/8qtra2s8ub#main-key requestID=81ca80mk04001d1heasg msg="OK: wrote 157B"
ноя 06 21:02:25 server gotosocial[129664]: timestamp="06/11/2024 21:02:25.261" func=server.init.func1.Logger.13.1 level=INFO latency="257.165µs" userAgent="Misskey/2024.10.1 (https://shitpost.poridge.club/)" method=GET statusCode=200 path=/fileserver/01QDFX9DAY02T3A3CJW67MB94V/attachment/original/01JBYDNDZV8XKR5JVCNNH8J1MX.jpeg clientIP=178.208.94.78 requestID=dnca80mk0400041ccsmg msg="OK: wrote 266kiB"
ноя 06 21:02:25 server gotosocial[129664]: timestamp="06/11/2024 21:02:25.262" func=server.init.func1.Logger.13.1 level=INFO latency="500.279µs" userAgent="Misskey/2024.10.1 (https://shitpost.poridge.club/)" method=GET statusCode=200 path=/fileserver/01QDFX9DAY02T3A3CJW67MB94V/attachment/original/01J5BW8XZWRZMDB36PX8N4X3QX.jpg clientIP=178.208.94.78 requestID=dsca80mk04000p2ypj8g msg="OK: wrote 696kiB"
ноя 06 21:02:36 server gotosocial[129664]: timestamp="06/11/2024 21:02:36.741" func=server.init.func1.Logger.13.1 level=INFO latency="284.411µs" userAgent=MoshidonAndroid/2.3.0+fork.107.moshinda-nightly+@2024-09-09 method=GET statusCode=200 path=/api/v1/accounts/01J6HXVT8RRK24YBSRZCT8B3E4 clientIP=192.168.50.1 requestID=8j2t80mk04001s7cpxz0 msg="OK: wrote 1.26kiB"
ноя 06 21:02:36 server gotosocial[129664]: timestamp="06/11/2024 21:02:36.765" func=server.init.func1.Logger.13.1 level=INFO latency="148.675µs" userAgent=MoshidonAndroid/2.3.0+fork.107.moshinda-nightly+@2024-09-09 method=GET statusCode=200 path=/api/v1/accounts/relationships clientIP=192.168.50.1 requestID=bp2t80mk0400055jy7wg msg="OK: wrote 182B"
ноя 06 21:02:36 server gotosocial[129664]: timestamp="06/11/2024 21:02:36.769" func=server.init.func1.Logger.13.1 level=INFO latency="4.139174ms" userAgent=MoshidonAndroid/2.3.0+fork.107.moshinda-nightly+@2024-09-09 method=GET statusCode=200 path=/api/v1/accounts/01J6HXVT8RRK24YBSRZCT8B3E4/statuses clientIP=192.168.50.1 requestID=bp2t80mk04000wvm4x1g msg="OK: wrote 2.62kiB"
ноя 06 21:02:36 server gotosocial[129664]: timestamp="06/11/2024 21:02:36.821" func=server.init.func1.Logger.13.1 level=INFO latency="526.791µs" userAgent=MoshidonAndroid/2.3.0+fork.107.moshinda-nightly+@2024-09-09 method=GET statusCode=200 path=/api/v1/accounts/01J6HXVT8RRK24YBSRZCT8B3E4/statuses clientIP=192.168.50.1 requestID=jp2t80mk040003qx5gfg msg="OK: wrote 26B"
ноя 06 21:02:36 server gotosocial[129664]: timestamp="06/11/2024 21:02:36.847" func=server.init.func1.Logger.13.1 level=INFO latency="1.792661ms" userAgent=MoshidonAndroid/2.3.0+fork.107.moshinda-nightly+@2024-09-09 method=GET statusCode=200 path=/api/v1/accounts/01J6HXVT8RRK24YBSRZCT8B3E4/statuses clientIP=192.168.50.1 requestID=np2t80mk0400031p9150 msg="OK: wrote 1.97kiB"
ноя 06 21:02:41 server gotosocial[129664]: timestamp="06/11/2024 21:02:41.258" func=server.init.func1.Logger.13.1 level=INFO latency="120.275083ms" userAgent=Uptime-Kuma/1.23.15 method=GET statusCode=200 path=/ clientIP=192.168.50.1 requestID=eaba80mk04001t14m6b0 msg="OK: wrote 15.9kiB"
ноя 06 21:02:48 server gotosocial[129664]: timestamp="06/11/2024 21:02:48.257" func=workers.(*Processor).ProcessFromClientAPI level=INFO activityType=Undo objectType=Follow fromAccount=sl msg="processing from client API"
ноя 06 21:02:48 server gotosocial[129664]: timestamp="06/11/2024 21:02:48.257" func=server.init.func1.Logger.13.1 level=INFO latency="1.450486ms" userAgent=MoshidonAndroid/2.3.0+fork.107.moshinda-nightly+@2024-09-09 method=POST statusCode=200 path=/api/v1/accounts/01J6HXVT8RRK24YBSRZCT8B3E4/unfollow clientIP=192.168.50.1 requestID=7ysa80mk04000q9h0c40 msg="OK: wrote 173B"
ноя 06 21:02:48 server gotosocial[129664]: timestamp="06/11/2024 21:02:48.258" func=federation.(*federatingActor).Send level=INFO msg="send activity Undo via outbox https://gts.sadium.cyou/users/sl/outbox"
ноя 06 21:02:48 server gotosocial[129664]: timestamp="06/11/2024 21:02:48.312" func=httpclient.(*Client).DoOnce level=INFO method=POST url=https://shitpost.poridge.club/inbox contentType="application/ld+json; profile=\"https://www.w3.org/ns/activitystreams\"" attempt="1" pubKeyID=https://gts.sadium.cyou/users/sl/main-key msg="202 Accepted"
ноя 06 21:02:49 server gotosocial[129664]: timestamp="06/11/2024 21:02:49.711" func=workers.(*Processor).ProcessFromClientAPI level=INFO activityType=Create objectType=Follow fromAccount=sl msg="processing from client API"
ноя 06 21:02:49 server gotosocial[129664]: timestamp="06/11/2024 21:02:49.711" func=server.init.func1.Logger.13.1 level=INFO latency="3.049ms" userAgent=MoshidonAndroid/2.3.0+fork.107.moshinda-nightly+@2024-09-09 method=POST statusCode=200 path=/api/v1/accounts/01J6HXVT8RRK24YBSRZCT8B3E4/follow clientIP=192.168.50.1 requestID=xjvt80mk04000hh2syt0 msg="OK: wrote 179B"
ноя 06 21:02:49 server gotosocial[129664]: timestamp="06/11/2024 21:02:49.711" func=federation.(*federatingActor).Send level=INFO msg="send activity Follow via outbox https://gts.sadium.cyou/users/sl/outbox"
ноя 06 21:02:49 server gotosocial[129664]: timestamp="06/11/2024 21:02:49.768" func=httpclient.(*Client).DoOnce level=INFO method=POST url=https://shitpost.poridge.club/inbox contentType="application/ld+json; profile=\"https://www.w3.org/ns/activitystreams\"" attempt="1" pubKeyID=https://gts.sadium.cyou/users/sl/main-key msg="202 Accepted"
ноя 06 21:02:49 server gotosocial[129664]: timestamp="06/11/2024 21:02:49.804" func=server.init.func1.Logger.13.1 level=INFO latency="3.42376ms" userAgent="Misskey/2024.10.1 (https://shitpost.poridge.club/)" method=GET statusCode=200 path=/users/sl clientIP=178.208.94.78 pubKeyID=https://shitpost.poridge.club/users/8qtra2s8ub#main-key requestID=92wa80mk040010em2vq0 msg="OK: wrote 2.13kiB"
ноя 06 21:02:49 server gotosocial[129664]: timestamp="06/11/2024 21:02:49.891" func=server.init.func1.Logger.13.1 level=INFO latency="674.74µs" userAgent="Misskey/2024.10.1 (https://shitpost.poridge.club/)" method=GET statusCode=200 path=/users/sl/following clientIP=178.208.94.78 pubKeyID=https://shitpost.poridge.club/users/8qtra2s8ub#main-key requestID=mawa80mk04000vp46xeg msg="OK: wrote 156B"
ноя 06 21:02:49 server gotosocial[129664]: timestamp="06/11/2024 21:02:49.891" func=server.init.func1.Logger.13.1 level=INFO latency="698.889µs" userAgent="Misskey/2024.10.1 (https://shitpost.poridge.club/)" method=GET statusCode=200 path=/users/sl/followers clientIP=178.208.94.78 pubKeyID=https://shitpost.poridge.club/users/8qtra2s8ub#main-key requestID=mawa80mk040014f2d3v0 msg="OK: wrote 157B"
ноя 06 21:02:49 server gotosocial[129664]: timestamp="06/11/2024 21:02:49.936" func=server.init.func1.Logger.13.1 level=INFO latency="2.186608ms" userAgent="Misskey/2024.10.1 (https://shitpost.poridge.club/)" method=GET statusCode=200 path=/fileserver/01QDFX9DAY02T3A3CJW67MB94V/attachment/original/01JBYDNDZV8XKR5JVCNNH8J1MX.jpeg clientIP=178.208.94.78 requestID=stwa80mk04001f9te3yg msg="OK: wrote 266kiB"
ноя 06 21:02:49 server gotosocial[129664]: timestamp="06/11/2024 21:02:49.936" func=server.init.func1.Logger.13.1 level=INFO latency="1.60944ms" userAgent="Misskey/2024.10.1 (https://shitpost.poridge.club/)" method=GET statusCode=200 path=/fileserver/01QDFX9DAY02T3A3CJW67MB94V/attachment/original/01J5BW8XZWRZMDB36PX8N4X3QX.jpg clientIP=178.208.94.78 requestID=sywa80mk04001w6rw3c0 msg="OK: wrote 696kiB"
ноя 06 21:03:01 server gotosocial[129664]: timestamp="06/11/2024 21:03:01.089" func=server.init.func1.Logger.13.1 level=INFO latency="20.458683ms" userAgent=MoshidonAndroid/2.3.0+fork.107.moshinda-nightly+@2024-09-09 method=GET statusCode=200 path=/api/v1/accounts/01J6HXVT8RRK24YBSRZCT8B3E4 clientIP=192.168.50.1 requestID=9fja80mk04001nraxamg msg="OK: wrote 1.26kiB"
ноя 06 21:03:01 server gotosocial[129664]: timestamp="06/11/2024 21:03:01.116" func=server.init.func1.Logger.13.1 level=INFO latency="5.39856ms" userAgent=MoshidonAndroid/2.3.0+fork.107.moshinda-nightly+@2024-09-09 method=GET statusCode=200 path=/api/v1/accounts/relationships clientIP=192.168.50.1 requestID=ezja80mk040007cs0tbg msg="OK: wrote 182B"
ноя 06 21:03:01 server gotosocial[129664]: timestamp="06/11/2024 21:03:01.135" func=server.init.func1.Logger.13.1 level=INFO latency="23.398488ms" userAgent=MoshidonAndroid/2.3.0+fork.107.moshinda-nightly+@2024-09-09 method=GET statusCode=200 path=/api/v1/accounts/01J6HXVT8RRK24YBSRZCT8B3E4/statuses clientIP=192.168.50.1 requestID=ezja80mk04000hw8mc90 msg="OK: wrote 2.62kiB"
ноя 06 21:03:01 server gotosocial[129664]: timestamp="06/11/2024 21:03:01.171" func=server.init.func1.Logger.13.1 level=INFO latency="656.728µs" userAgent=MoshidonAndroid/2.3.0+fork.107.moshinda-nightly+@2024-09-09 method=GET statusCode=200 path=/api/v1/accounts/01J6HXVT8RRK24YBSRZCT8B3E4/statuses clientIP=192.168.50.1 requestID=pbja80mk04001c75en20 msg="OK: wrote 26B"
ноя 06 21:03:01 server gotosocial[129664]: timestamp="06/11/2024 21:03:01.188" func=server.init.func1.Logger.13.1 level=INFO latency="2.54927ms" userAgent=MoshidonAndroid/2.3.0+fork.107.moshinda-nightly+@2024-09-09 method=GET statusCode=200 path=/api/v1/accounts/01J6HXVT8RRK24YBSRZCT8B3E4/statuses clientIP=192.168.50.1 requestID=rbja80mk04000ya81y10 msg="OK: wrote 1.97kiB"
ноя 06 21:03:08 server gotosocial[129664]: timestamp="06/11/2024 21:03:08.612" func=server.init.func1.Logger.13.1 level=INFO latency="572.334µs" userAgent=MoshidonAndroid/2.3.0+fork.107.moshinda-nightly+@2024-09-09 method=GET statusCode=200 path=/api/v1/markers clientIP=192.168.50.1 requestID=rg0ta0mk04001s7v67cg msg="OK: wrote 181B"
ноя 06 21:03:08 server gotosocial[129664]: timestamp="06/11/2024 21:03:08.618" func=server.init.func1.Logger.13.1 level=INFO latency="581.767µs" userAgent=MoshidonAndroid/2.3.0+fork.107.moshinda-nightly+@2024-09-09 method=GET statusCode=200 path=/api/v1/markers clientIP=192.168.50.1 requestID=s40ta0mk04000029n5c0 msg="OK: wrote 181B"
ноя 06 21:03:08 server gotosocial[129664]: timestamp="06/11/2024 21:03:08.631" func=server.init.func1.Logger.13.1 level=INFO latency="27.30563ms" userAgent="Misskey/2024.10.1 (https://shitpost.poridge.club/)" method=GET statusCode=200 path=/users/sl clientIP=178.208.94.78 pubKeyID=https://shitpost.poridge.club/users/8qtra2s8ub#main-key requestID=qc0ta0mk04001kn8rd50 msg="OK: wrote 2.13kiB"
ноя 06 21:03:08 server gotosocial[129664]: timestamp="06/11/2024 21:03:08.672" func=bundb.(*notificationDB).GetNotificationsByIDs.func3 level=ERROR requestID=rc0ta0mk040002kc9zm0 msg="error populating notif 01JBYTADD4T4NX0A6MN7Z4VHY4: PopulateNotification: error populating notif status: sql: no rows in result set"
ноя 06 21:03:08 server gotosocial[129664]: timestamp="06/11/2024 21:03:08.681" func=server.init.func1.Logger.13.1 level=INFO latency="75.752566ms" userAgent=MoshidonAndroid/2.3.0+fork.107.moshinda-nightly+@2024-09-09 method=GET statusCode=200 path=/api/v1/timelines/home clientIP=192.168.50.1 requestID=qm0ta0mk04000ysqsdd0 msg="OK: wrote 1.75kiB"
ноя 06 21:03:08 server gotosocial[129664]: timestamp="06/11/2024 21:03:08.720" func=server.init.func1.Logger.13.1 level=INFO latency="1.073737ms" userAgent="Misskey/2024.10.1 (https://shitpost.poridge.club/)" method=GET statusCode=200 path=/users/sl/following clientIP=178.208.94.78 pubKeyID=https://shitpost.poridge.club/users/8qtra2s8ub#main-key requestID=5w1aa0mk0400101rjby0 msg="OK: wrote 156B"
ноя 06 21:03:08 server gotosocial[129664]: timestamp="06/11/2024 21:03:08.720" func=server.init.func1.Logger.13.1 level=INFO latency="1.200439ms" userAgent="Misskey/2024.10.1 (https://shitpost.poridge.club/)" method=GET statusCode=200 path=/users/sl/followers clientIP=178.208.94.78 pubKeyID=https://shitpost.poridge.club/users/8qtra2s8ub#main-key requestID=5w1aa0mk040005qj83mg msg="OK: wrote 157B"
ноя 06 21:03:08 server gotosocial[129664]: timestamp="06/11/2024 21:03:08.749" func=server.init.func1.Logger.13.1 level=INFO latency="138.112332ms" userAgent=MoshidonAndroid/2.3.0+fork.107.moshinda-nightly+@2024-09-09 method=GET statusCode=200 path=/api/v1/notifications clientIP=192.168.50.1 requestID=rc0ta0mk040002kc9zm0 msg="OK: wrote 11.2kiB"
ноя 06 21:03:08 server gotosocial[129664]: timestamp="06/11/2024 21:03:08.765" func=server.init.func1.Logger.13.1 level=INFO latency="1.119938ms" userAgent="Misskey/2024.10.1 (https://shitpost.poridge.club/)" method=GET statusCode=200 path=/fileserver/01QDFX9DAY02T3A3CJW67MB94V/attachment/original/01JBYDNDZV8XKR5JVCNNH8J1MX.jpeg clientIP=178.208.94.78 requestID=bg1aa0mk04001mx14hpg msg="OK: wrote 266kiB"
ноя 06 21:03:08 server gotosocial[129664]: timestamp="06/11/2024 21:03:08.768" func=server.init.func1.Logger.13.1 level=INFO latency="1.388941ms" userAgent="Misskey/2024.10.1 (https://shitpost.poridge.club/)" method=GET statusCode=200 path=/fileserver/01QDFX9DAY02T3A3CJW67MB94V/attachment/original/01J5BW8XZWRZMDB36PX8N4X3QX.jpg clientIP=178.208.94.78 requestID=bw1aa0mk04001wtnavbg msg="OK: wrote 696kiB"
ноя 06 21:03:08 server gotosocial[129664]: timestamp="06/11/2024 21:03:08.894" func=bundb.(*notificationDB).GetNotificationsByIDs.func3 level=ERROR requestID=pr1aa0mk04000yczydf0 msg="error populating notif 01JANV7QVW24W9XQG6RBGXCVWX: PopulateNotification: error populating notif status: sql: no rows in result set"
ноя 06 21:03:09 server gotosocial[129664]: timestamp="06/11/2024 21:03:09.020" func=server.init.func1.Logger.13.1 level=INFO latency="165.85444ms" userAgent=MoshidonAndroid/2.3.0+fork.107.moshinda-nightly+@2024-09-09 method=GET statusCode=200 path=/api/v1/notifications clientIP=192.168.50.1 requestID=pr1aa0mk04000yczydf0 msg="OK: wrote 11.7kiB"
ноя 06 21:03:16 server gotosocial[129664]: timestamp="06/11/2024 21:03:16.074" func=workers.(*Processor).ProcessFromClientAPI level=INFO activityType=Undo objectType=Follow fromAccount=sl msg="processing from client API"
ноя 06 21:03:16 server gotosocial[129664]: timestamp="06/11/2024 21:03:16.075" func=server.init.func1.Logger.13.1 level=INFO latency="2.591673ms" userAgent=MoshidonAndroid/2.3.0+fork.107.moshinda-nightly+@2024-09-09 method=POST statusCode=200 path=/api/v1/accounts/01J6HXVT8RRK24YBSRZCT8B3E4/unfollow clientIP=192.168.50.1 requestID=x0faa0mk04001k2x5zag msg="OK: wrote 173B"
ноя 06 21:03:16 server gotosocial[129664]: timestamp="06/11/2024 21:03:16.112" func=federation.(*federatingActor).Send level=INFO msg="send activity Undo via outbox https://gts.sadium.cyou/users/sl/outbox"
ноя 06 21:03:16 server gotosocial[129664]: timestamp="06/11/2024 21:03:16.186" func=httpclient.(*Client).DoOnce level=INFO method=POST url=https://shitpost.poridge.club/inbox contentType="application/ld+json; profile=\"https://www.w3.org/ns/activitystreams\"" attempt="1" pubKeyID=https://gts.sadium.cyou/users/sl/main-key msg="202 Accepted"
ноя 06 21:03:17 server gotosocial[129664]: timestamp="06/11/2024 21:03:17.522" func=workers.(*Processor).ProcessFromClientAPI level=INFO activityType=Create objectType=Follow fromAccount=sl msg="processing from client API"
ноя 06 21:03:17 server gotosocial[129664]: timestamp="06/11/2024 21:03:17.522" func=server.init.func1.Logger.13.1 level=INFO latency="2.233171ms" userAgent=MoshidonAndroid/2.3.0+fork.107.moshinda-nightly+@2024-09-09 method=POST statusCode=200 path=/api/v1/accounts/01J6HXVT8RRK24YBSRZCT8B3E4/follow clientIP=192.168.50.1 requestID=j0jaa0mk04000nf0j0f0 msg="OK: wrote 179B"
ноя 06 21:03:17 server gotosocial[129664]: timestamp="06/11/2024 21:03:17.523" func=federation.(*federatingActor).Send level=INFO msg="send activity Follow via outbox https://gts.sadium.cyou/users/sl/outbox"
ноя 06 21:03:17 server gotosocial[129664]: timestamp="06/11/2024 21:03:17.617" func=httpclient.(*Client).DoOnce level=INFO method=POST url=https://shitpost.poridge.club/inbox contentType="application/ld+json; profile=\"https://www.w3.org/ns/activitystreams\"" attempt="1" pubKeyID=https://gts.sadium.cyou/users/sl/main-key msg="202 Accepted"
```

</p>
</details> 

### What you expected to happen?

Instant delivery of approval request after first request

### How to reproduce it?

Try to get pending approval request for locked Misskey (or it fork) account

**Comment by tsmethurst** [comment]
Hmm, those logs you posted all show that the Follow is being POSTed to the inbox of the followee immediately after you make the request (within a matter of milliseconds), so I'm not sure how to follow up on this. Do you have logs where it's taking longer?

**Comment by SadmL** [comment]
No, I'm also noticed that in logs all looks like happens instantly, but in fact it's not.

https://github.com/user-attachments/assets/a3bc4554-136e-4083-beda-619b5548670e

**Comment by tsmethurst** [comment]
Since GtS is sending out those POSTs and getting a 202 Accepted back almost immediately, I think any lag in processing + notification that you're seeing must be from the Misskey side. Once we've sent out the message to their inbox and they've returned a 202 response (as shown in the logs), it's up to them to process the message and create notifications etc. The videos show that you're waiting for a notification to show up on the *key side, but there's nothing we can do about that.

**Comment by SadmL** [comment]
Closing then, I guess...

**Comment by CDN18** [comment]
Today I encountered this issue as well. TLDR: This problem is likely related to the shared inbox.

After setting up a Misskey instance and troubleshooting, I noticed the issue only affects a specific situation of follows: when a GoToSocial user tries to follow a Misskey user who **hasn't followed them back before** and **has manual approval for follow requests enabled**. The Misskey user won't receive the follow request. From the GoToSocial user's side, the request will remain pending indefinitely.

In the Misskey instance's Job Queue, a delayed job appears which, after exceeding the maximum retry attempts, I believe goes into failed job list.

The Job Data for the follow request is as follows:

```
{
  "jobData": {
    "activity": {
      "@context": "https://www.w3.org/ns/activitystreams",
      "actor": "https://scg.owu.one/users/icarus",
      "id": "https://scg.owu.one/users/icarus/follow/0184ZATKEVN6PAF2WP202XMKZ9",
      "object": "https://shk.owu.one/users/a0e340gpzf7b0001",
      "to": "https://shk.owu.one/users/a0e340gpzf7b0001",
      "type": "Follow"
    },
    "signature": {
      "scheme": "Signature",
      "params": {
        "keyId": "https://scg.owu.one/users/icarus/main-key",
        "algorithm": "hs2019",
        "headers": [
          "(request-target)",
          "host",
          "date",
          "digest"
        ],
        "signature": "UvmELdIe3E6t0BwZ602wfid1fxNUl37MQ7GzwPKVx9+GlgBG44FFdTjmzGL9Q7uWukEbgRe0O53UWp0Jna2oip2RZ3hNV1JdX2hIP2hNu2yAXR3RHgVPnM16VpQGckASTaEiukMFyI0ls5oZiV1j64h8wBBJFDEmCZeaipf/0EiY6Iz6o3dSXWWvVnn5aZx5ONTlbfN/y243s5uXAYluhbxzjQk/9HTePi2A1wcJzYmx3is4HmOs/pdcqptbkOSx0LH7rWr+sbdyLfzAuY5zLaQjVjTOjaAP6PAMX1WLzqB+/Hy1aXuk7P+gJFozstOmkk8/IRmSDs9b98IrTjTP1Q=="
      },
      "signingString": "(request-target): post /inbox\nhost: shk.owu.one\ndate: Sat, 09 Nov 2024 18:53:56 GMT\ndigest: SHA-256=7+wAKCV+LPtDUHromIknMf9M78gIFaOte1B4WOTvtWY=",
      "algorithm": "HS2019",
      "keyId": "https://scg.owu.one/users/icarus/main-key"
    }
  },
  "returnValue": null
}
```

The error message from misskey side is: 
```
UpdateValuesMissingError: Cannot perform update query because update values are not defined. Call "qb.set(...)" method to specify updated values.
    at UpdateQueryBuilder.createUpdateExpression (/sharkey/node_modules/.pnpm/typeorm@0.3.20_ioredis@5.4.1_pg@8.13.0/node_modules/typeorm/query-builder/UpdateQueryBuilder.js:430:19)
    at UpdateQueryBuilder.getQuery (/sharkey/node_modules/.pnpm/typeorm@0.3.20_ioredis@5.4.1_pg@8.13.0/node_modules/typeorm/query-builder/UpdateQueryBuilder.js:34:21)
    at UpdateQueryBuilder.getQueryAndParameters (/sharkey/node_modules/.pnpm/typeorm@0.3.20_ioredis@5.4.1_pg@8.13.0/node_modules/typeorm/query-builder/QueryBuilder.js:262:28)
    at UpdateQueryBuilder.execute (/sharkey/node_modules/.pnpm/typeorm@0.3.20_ioredis@5.4.1_pg@8.13.0/node_modules/typeorm/query-builder/UpdateQueryBuilder.js:81:50)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async ApPersonService.updatePerson (file:///sharkey/packages/backend/built/core/activitypub/models/ApPersonService.js:516:9)
    at async AccountMoveService.validateAlsoKnownAs (file:///sharkey/packages/backend/built/core/AccountMoveService.js:334:17)
    at async UserFollowingService.follow (file:///sharkey/packages/backend/built/core/UserFollowingService.js:175:32)
    at async ApInboxService.follow (file:///sharkey/packages/backend/built/core/activitypub/ApInboxService.js:193:9)
    at async ApInboxService.performOneActivity (file:///sharkey/packages/backend/built/core/activitypub/ApInboxService.js:159:20)
    at async ApInboxService.performActivity (file:///sharkey/packages/backend/built/core/activitypub/ApInboxService.js:138:22)
    at async InboxProcessorService.process (file:///sharkey/packages/backend/built/queue/processors/InboxProcessorService.js:193:28)
    at async Worker.processJob (/sharkey/node_modules/.pnpm/bullmq@5.13.2/node_modules/bullmq/dist/cjs/classes/worker.js:455:28)
    at async Worker.retryIfFailed (/sharkey/node_modules/.pnpm/bullmq@5.13.2/node_modules/bullmq/dist/cjs/classes/worker.js:640:24)
UpdateValuesMissingError: Cannot perform update query because update values are not defined. Call "qb.set(...)" method to specify updated values.
    at UpdateQueryBuilder.createUpdateExpression (/sharkey/node_modules/.pnpm/typeorm@0.3.20_ioredis@5.4.1_pg@8.13.0/node_modules/typeorm/query-builder/UpdateQueryBuilder.js:430:19)
    at UpdateQueryBuilder.getQuery (/sharkey/node_modules/.pnpm/typeorm@0.3.20_ioredis@5.4.1_pg@8.13.0/node_modules/typeorm/query-builder/UpdateQueryBuilder.js:34:21)
    at UpdateQueryBuilder.getQueryAndParameters (/sharkey/node_modules/.pnpm/typeorm@0.3.20_ioredis@5.4.1_pg@8.13.0/node_modules/typeorm/query-builder/QueryBuilder.js:262:28)
    at UpdateQueryBuilder.execute (/sharkey/node_modules/.pnpm/typeorm@0.3.20_ioredis@5.4.1_pg@8.13.0/node_modules/typeorm/query-builder/UpdateQueryBuilder.js:81:50)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async ApPersonService.updatePerson (file:///sharkey/packages/backend/built/core/activitypub/models/ApPersonService.js:516:9)
    at async AccountMoveService.validateAlsoKnownAs (file:///sharkey/packages/backend/built/core/AccountMoveService.js:334:17)
    at async UserFollowingService.follow (file:///sharkey/packages/backend/built/core/UserFollowingService.js:175:32)
    at async ApInboxService.follow (file:///sharkey/packages/backend/built/core/activitypub/ApInboxService.js:193:9)
    at async ApInboxService.performOneActivity (file:///sharkey/packages/backend/built/core/activitypub/ApInboxService.js:159:20)
    at async ApInboxService.performActivity (file:///sharkey/packages/backend/built/core/activitypub/ApInboxService.js:138:22)
    at async InboxProcessorService.process (file:///sharkey/packages/backend/built/queue/processors/InboxProcessorService.js:193:28)
    at async Worker.processJob (/sharkey/node_modules/.pnpm/bullmq@5.13.2/node_modules/bullmq/dist/cjs/classes/worker.js:455:28)
    at async Worker.retryIfFailed (/sharkey/node_modules/.pnpm/bullmq@5.13.2/node_modules/bullmq/dist/cjs/classes/worker.js:640:24)
UpdateValuesMissingError: Cannot perform update query because update values are not defined. Call "qb.set(...)" method to specify updated values.
    at UpdateQueryBuilder.createUpdateExpression (/sharkey/node_modules/.pnpm/typeorm@0.3.20_ioredis@5.4.1_pg@8.13.0/node_modules/typeorm/query-builder/UpdateQueryBuilder.js:430:19)
    at UpdateQueryBuilder.getQuery (/sharkey/node_modules/.pnpm/typeorm@0.3.20_ioredis@5.4.1_pg@8.13.0/node_modules/typeorm/query-builder/UpdateQueryBuilder.js:34:21)
    at UpdateQueryBuilder.getQueryAndParameters (/sharkey/node_modules/.pnpm/typeorm@0.3.20_ioredis@5.4.1_pg@8.13.0/node_modules/typeorm/query-builder/QueryBuilder.js:262:28)
    at UpdateQueryBuilder.execute (/sharkey/node_modules/.pnpm/typeorm@0.3.20_ioredis@5.4.1_pg@8.13.0/node_modules/typeorm/query-builder/UpdateQueryBuilder.js:81:50)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async ApPersonService.updatePerson (file:///sharkey/packages/backend/built/core/activitypub/models/ApPersonService.js:516:9)
    at async AccountMoveService.validateAlsoKnownAs (file:///sharkey/packages/backend/built/core/AccountMoveService.js:334:17)
    at async UserFollowingService.follow (file:///sharkey/packages/backend/built/core/UserFollowingService.js:175:32)
    at async ApInboxService.follow (file:///sharkey/packages/backend/built/core/activitypub/ApInboxService.js:193:9)
    at async ApInboxService.performOneActivity (file:///sharkey/packages/backend/built/core/activitypub/ApInboxService.js:159:20)
    at async ApInboxService.performActivity (file:///sharkey/packages/backend/built/core/activitypub/ApInboxService.js:138:22)
    at async InboxProcessorService.process (file:///sharkey/packages/backend/built/queue/processors/InboxProcessorService.js:193:28)
    at async Worker.processJob (/sharkey/node_modules/.pnpm/bullmq@5.13.2/node_modules/bullmq/dist/cjs/classes/worker.js:455:28)
    at async Worker.retryIfFailed (/sharkey/node_modules/.pnpm/bullmq@5.13.2/node_modules/bullmq/dist/cjs/classes/worker.js:640:24)
UpdateValuesMissingError: Cannot perform update query because update values are not defined. Call "qb.set(...)" method to specify updated values.
    at UpdateQueryBuilder.createUpdateExpression (/sharkey/node_modules/.pnpm/typeorm@0.3.20_ioredis@5.4.1_pg@8.13.0/node_modules/typeorm/query-builder/UpdateQueryBuilder.js:430:19)
    at UpdateQueryBuilder.getQuery (/sharkey/node_modules/.pnpm/typeorm@0.3.20_ioredis@5.4.1_pg@8.13.0/node_modules/typeorm/query-builder/UpdateQueryBuilder.js:34:21)
    at UpdateQueryBuilder.getQueryAndParameters (/sharkey/node_modules/.pnpm/typeorm@0.3.20_ioredis@5.4.1_pg@8.13.0/node_modules/typeorm/query-builder/QueryBuilder.js:262:28)
    at UpdateQueryBuilder.execute (/sharkey/node_modules/.pnpm/typeorm@0.3.20_ioredis@5.4.1_pg@8.13.0/node_modules/typeorm/query-builder/UpdateQueryBuilder.js:81:50)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async ApPersonService.updatePerson (file:///sharkey/packages/backend/built/core/activitypub/models/ApPersonService.js:516:9)
    at async AccountMoveService.validateAlsoKnownAs (file:///sharkey/packages/backend/built/core/AccountMoveService.js:334:17)
    at async UserFollowingService.follow (file:///sharkey/packages/backend/built/core/UserFollowingService.js:175:32)
    at async ApInboxService.follow (file:///sharkey/packages/backend/built/core/activitypub/ApInboxService.js:193:9)
    at async ApInboxService.performOneActivity (file:///sharkey/packages/backend/built/core/activitypub/ApInboxService.js:159:20)
    at async ApInboxService.performActivity (file:///sharkey/packages/backend/built/core/activitypub/ApInboxService.js:138:22)
    at async InboxProcessorService.process (file:///sharkey/packages/backend/built/queue/processors/InboxProcessorService.js:193:28)
    at async Worker.processJob (/sharkey/node_modules/.pnpm/bullmq@5.13.2/node_modules/bullmq/dist/cjs/classes/worker.js:455:28)
    at async Worker.retryIfFailed (/sharkey/node_modules/.pnpm/bullmq@5.13.2/node_modules/bullmq/dist/cjs/classes/worker.js:640:24)
UpdateValuesMissingError: Cannot perform update query because update values are not defined. Call "qb.set(...)" method to specify updated values.
    at UpdateQueryBuilder.createUpdateExpression (/sharkey/node_modules/.pnpm/typeorm@0.3.20_ioredis@5.4.1_pg@8.13.0/node_modules/typeorm/query-builder/UpdateQueryBuilder.js:430:19)
    at UpdateQueryBuilder.getQuery (/sharkey/node_modules/.pnpm/typeorm@0.3.20_ioredis@5.4.1_pg@8.13.0/node_modules/typeorm/query-builder/UpdateQueryBuilder.js:34:21)
    at UpdateQueryBuilder.getQueryAndParameters (/sharkey/node_modules/.pnpm/typeorm@0.3.20_ioredis@5.4.1_pg@8.13.0/node_modules/typeorm/query-builder/QueryBuilder.js:262:28)
    at UpdateQueryBuilder.execute (/sharkey/node_modules/.pnpm/typeorm@0.3.20_ioredis@5.4.1_pg@8.13.0/node_modules/typeorm/query-builder/UpdateQueryBuilder.js:81:50)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async ApPersonService.updatePerson (file:///sharkey/packages/backend/built/core/activitypub/models/ApPersonService.js:516:9)
    at async AccountMoveService.validateAlsoKnownAs (file:///sharkey/packages/backend/built/core/AccountMoveService.js:334:17)
    at async UserFollowingService.follow (file:///sharkey/packages/backend/built/core/UserFollowingService.js:175:32)
    at async ApInboxService.follow (file:///sharkey/packages/backend/built/core/activitypub/ApInboxService.js:193:9)
    at async ApInboxService.performOneActivity (file:///sharkey/packages/backend/built/core/activitypub/ApInboxService.js:159:20)
    at async ApInboxService.performActivity (file:///sharkey/packages/backend/built/core/activitypub/ApInboxService.js:138:22)
    at async InboxProcessorService.process (file:///sharkey/packages/backend/built/queue/processors/InboxProcessorService.js:193:28)
    at async Worker.processJob (/sharkey/node_modules/.pnpm/bullmq@5.13.2/node_modules/bullmq/dist/cjs/classes/worker.js:455:28)
    at async Worker.retryIfFailed (/sharkey/node_modules/.pnpm/bullmq@5.13.2/node_modules/bullmq/dist/cjs/classes/worker.js:640:24)
```

Checking `sharkey/packages/backend/built/core/activitypub/models/ApPersonService.js:516`, the function looks like this: 
```javascript
await this.followingsRepository.update({ // line 516
            followerId: exist.id
        }, {
            followerSharedInbox: person.sharedInbox ?? person.endpoints?.sharedInbox
        });
```

This corresponds to: https://github.com/misskey-dev/misskey/blob/develop/packages/backend/src/core/activitypub/models/ApPersonService.ts#L596

I think Misskey requires all target accounts to have a shared inbox when executing this function. However, GoToSocial hasn't implemented shared inbox yet, as seen in #1894.

**Comment by CDN18** [comment]
I've submitted the corresponding fix PR, and it's been merged into Misskey. If the Misskey instance you want to follow upgrades to version 2024.11 (which I personally estimate will be when this version is released) or cherry-picks this commit, then the issue should no longer occur.

Also, about nine months ago, when the migration feature of GoToSocial was newly implemented, I migrated and noticed that some of my Misskey followers didn't get transferred. This issue was probably due to the same reason.

**Comment by tsmethurst** [comment]
Nice one :)
